README.md

```diff
@@ -63,12 +63,12 @@ dependencies[] = restful
  }
 ```
 
-####restful\_custom/plugins/restful/myplugin.inc
+####restful\_custom/plugins/restful/thenameoftheplugin.inc
```

Comment:
maybe just change this to ``restful\_custom/plugins/restful/articles.inc``, and keep the comment about the plugin name == the ``name`` property?